### PR TITLE
FIX: bug in edb configuration padstack

### PR DESCRIPTION
**Pull request by ring630**

### CODE REVIEW

**Review by svandenb-dev — DISMISSED** [review]
LGTM

File: src/pyedb/configuration/cfg_boundaries.py [hunk 1]
```diff
@@ -97,19 +97,25 @@ def apply(self):
...
+            self._pedb.hfss.hfss_extent_info.radiation_level = self.pml_radiation_factor
         self._pedb.hfss.hfss_extent_info.extent_type = self.dielectric_extents_type.name.lower()
         if self.dielectric_base_polygon:
             self._pedb.hfss.hfss_extent_info.dielectric_base_polygon = self.dielectric_base_polygon
-        self._pedb.hfss.hfss_extent_info.dielectric_extent_size = float(self.horizontal_padding)
+        if self.horizontal_padding:
+            self._pedb.hfss.hfss_extent_info.dielectric_extent_size = float(self.horizontal_padding)
         self._pedb.hfss.hfss_extent_info.honor_user_dielectric = self.honor_primitives_on_dielectric_layers
         self._pedb.hfss.hfss_extent_info.extent_type = self.air_box_extents_type.name.lower()
         self._pedb.hfss.hfss_extent_info.truncate_air_box_at_ground = self.air_box_truncate_model_ground_layers
```
Comment (ring630) [inline src/pyedb/configuration/cfg_boundaries.py:111]: @svandenb-dev You forget these ones?

Comment (svandenb-dev) [reply src/pyedb/configuration/cfg_boundaries.py:111]: These are boolean you want also to don't touch ?

Comment (ring630) [reply src/pyedb/configuration/cfg_boundaries.py]: @svandenb-dev  Boolean are also attributes. Do not touch.

**Review by svandenb-dev — APPROVED** [review]
LGTM

**Pull request merged**